modules/openid_auth/app/controllers/openid_auth/v0/validation_controller.rb

```diff
@@ -1,18 +1,27 @@
 # frozen_string_literal: true
 
 require_dependency 'openid_auth/application_controller'
+require 'awesome_print'
```

Comment:
Was this left in here for some debugging purpose?

Reply:
Removed